en/docs/integrations/build-your-first-integration.md

```diff
@@ -0,0 +1,59 @@
+# Build Your First Integration
+Choreo allows you to efficiently build integrations by connecting APIs, events, and streams. Choreo also provides a comprehensive set of prebuilt integration templates that you can select and use depending on your requirement.
+
+This quick start guide walks you through the steps to quickly configure and try out the **GitHub New Issue to Google Sheets Row** integration in Choreo.
+
+## Prerequisites:
+- A GitHub account to connect to in the integration configuration.
+- A Google account to connect to in the integration configuration.
+- A blank spreadsheet named `Sample GitHub Integration` created in the Google account. This is the spreadsheet name to use in the integration configuration. 
+  The spreadsheet will have a default worksheet named `Sheet1`. This is the worksheet name to use in the integration configuration.
```

Comment:
Shall we avoid future tense and replace "will have" with "includes"?

Reply:
Updated the prerequisites section.